### DISCUSSION TITLE
Filament Runout Sensor Connected to Octopus

### DISCUSSION THREAD

**Discussion by fiveorders** [bigtreetech/BIGTREETECH-TouchScreenFirmware#2863] [General]
Hello, is there a way to make LCD Feature Settings to control the Filament Sensor On/Off which is connected to the Octopus Pro instead of TFT?

**Comment by kisslorand** [comment]
Settings -> Feature -> Filament sensor

**Reply by fiveorders** [reply]
Thanks but it doesnt control the filament sensor, I set it off but FS keeps functioning

**Comment by kisslorand** [comment]
Ah! Don't send M75 to Marlin, than it will ignore the FS and will not report its status

**Comment by fiveorders** [comment]
How will I not sent send m75 to marlin 🙂 I mean I am not sending anything?

**Comment by kisslorand** [comment]
Marlin takes the FS into account only if it knows that there's an ongoing print job. M75 is the command to let Marlin know that a print job is started.

**Comment by fiveorders** [comment]
ah ok you mean remove it from the start gcode. but in that case, will the filament sensor work and will I be able to control sensor on/off through features/Filament sensor setting?

**Comment by kisslorand** [comment]
No.

**Reply by fiveorders** [reply]
So what was your point for advising removing M75 from start gcode? My purpose was disabling filament sensor during print

**Comment by kisslorand** [comment]
My bad, with removing M75 you disable the FS _before_ the start of the print job. You can try disabling FS _during_ a print job by sending `M77` to Marlin.

**Comment by fiveorders** [comment]
Alright thank you
